### ISSUE TITLE
m65 rgb pro not detected

### ISSUE THREAD

**Issue by TheChellgames** [ccMSC/ckb#415]
I realize that its not under the supported devices, but I am wondering if it will be soon. I can imagine its not an easy process but i would greatly appreciate it. 

Thank you for this amazing software!

**Comment by tatokis** [comment]
There is experimental support for the M65 RGB Pro on the newdev branch.

**Comment by TheChellgames** [comment]
@tatokis How do I go about accessing that?

**Comment by TheChellgames** [comment]
@tatokis how do i find that?

**Comment by lukeberry99** [comment]
@TheChellgames there are currently no downloadable binaries, you'll have to compile it yourself.

**Comment by TheChellgames** [comment]
@lukeberry99 Can you or someone walk me through or help me do that please

**Comment by andrew904** [comment]
@lukeberry99 Having the same problems as @TheChellgames and need a walkthrough, I'm quite new.

**Comment by lmansur** [comment]
## This is how i did in Ubuntu:

### Uninstall previous ckb version:
For Ubuntu 15.04 and above
```
sudo systemctl stop ckb-daemon
sudo rm -f /usr/lib/systemd/system/ckb-daemon.service
```
For Ubuntu earlier than 15.04
```
sudo service ckb-daemon stop
sudo rm -f /etc/init/ckb-daemon.conf
```

Now do this regardless of your Ubuntu version
```
sudo rm -f /usr/bin/ckb /usr/bin/ckb-daemon /usr/share/applications/ckb.desktop /usr/share/icons/hicolor/512x512/apps/ckb.png
sudo rm -rf /usr/bin/ckb-animations
```
Your old ckb is uninstalled.
### Installing new version
Download/clone [this branch](https://github.com/ccMSC/ckb/tree/newdev) and extract it somewhere.
Move to the extracted location and double click _quickinstall_ file ( or do the equivalent in terminal ). This will install the new branch and M65 Pro should work.

The process should be similar in OSX, just follow the documentation.

**Comment by schneems** [comment]
According to #356 support for the pro model hasn't been implemented yet

**Comment by schneems** [comment]
Never mind, was only looking at master, didn't check `newdev` thanks!

**Comment by LxFx** [comment]
Happy user of the newdev branch for my M65 RGB Pro. When will this branch be merged with the master branch?

**Comment by ags911** [comment]
Hey LxFx, is it working for you? I have the same build but it's not being detected. Do you mind sharing the zip here to try please?

**Comment by LxFx** [comment]
Hi, yes everything works. I'm on Arch Linux. I just cloned the newdev branch and adapted the git PKGBUILD from the AUR and compiled successfully. I can share the PKGBUILD if that helps?

**Comment by ags911** [comment]
That would be amazing bud thanks. I'm on OSX though would it work?

**Comment by LxFx** [comment]
I'm quite sure it won't. But nevertheless, here goes...

https://drive.google.com/file/d/0B98pOitK_uzlSk5XQnA0cEptRWM/view?usp=sharing

**Comment by ags911** [comment]
Thanks, I'll give it a try.

**Comment by chrylis** [comment]
I have the Pro and am not seeing it as of the latest AUR package, which is geb1fe4e. @LxFx, do you know if the `ckb-git-latest` version includes this support?

**Comment by LxFx** [comment]
No, you'll need to clone the newdev branch and use my PKGBUILD linked above and ckb-git.install from the AUR to build it. Should still work.

By the way, you can save your favorite profile/config in the mouse itself. So you don't really need this software anymore, unless you want to change the LED theme from time to time.
